index.html

```diff
@@ -1,118 +1,114 @@
-				<ul>
-					<li>
-						<h3>небольшая площадь</h3>
-						<p>— №3 —<br> 
-							все достопримечательности<br> 
-						находятся очень близко</p>
-					</li>
-					<li>
-						<h3>красивая дорога</h3>
-						<p>— №4 —<br>
-						ехать в седону из лас-вегаса совсем<br> 
-						не скучно!</p>
-					</li>
-					<li>
-						<h3>мало туристов</h3>
-						<p>— №5 —<br> 
-						большинство едет в гранд каньон<br> 
-						и толпится там</p>
-					</li>
-				</ul>
-  			</section>
+  <section>
+    <ul>
+     <li>					
+      <img src="img/icon-home.svg" width="75" height="72" alt="жилье">	
+      <h2>Жильё</h2>
+      <p>Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+    </li>
+    <li>
+      <img src="img/icon-food.svg" width="75" height="70" alt="еда">
+      <h2>Еда</h2>
+      <p>Всегда заказывайте фирменный бургер, вы не разочаруетесь!</p>
+    </li>
+    <li>
+      <img src="img/icon-gift.svg" width="64" height="77" alt="сувениры">
+      <h2>Сувениры</h2>
+      <p>Не только китайского, но и местного производства!</p>
+    </li>
+  </ul>	
+</section>
 
-  			<section>
-  				<h2>Поиск гостиницы в Седоне</h2>
-  				<p><b>Заинтересовались?</b>  <br>
-  				Укажите предполагаемые даты поездки,<br>
-  				  и мы покажем вам лучшие предложения гостиниц в седоне  </p>
-  				 <!-- Здесь будет форма поиска гостиниц --> 
-  			</section>
-  		</main>
+<section>
+  <h2>Там есть  Мост<br> дьявола</h2>
+  <p>— №2 —</p>
+  <p>Да, по нему можно пройти! Если конечно вы осмелитесь</p>
+  <img src="https://via.placeholder.com/800x256" alt="Каменный мост Дьявола над пропастью">
+</section>
 
-  		<footer>
-  			<section>
-  				<h2>Контакты</h2>
-  				<p>#visitSEDONA<br>
-  				Тел: <a href="tel:+74956660166">+7 (495) 666-01-66</a>
-  				</p>
-  			</section>
+<section> 
+  <ul>
+   <li>
+    <h2>небольшая площадь</h2>
+    <p>— №3 — все достопримечательности находятся очень близко</p>
+  </li>
+  <li>
+    <h2>красивая дорога</h2>
+    <p>— №4 — ехать в седону из лас-вегаса совсем
+    не скучно!</p>
+  </li>
+  <li>
+    <h2>мало туристов</h2>
+    <p>— №5 — большинство едет в гранд каньон и толпится там</p>
+  </li>
+</ul>
+</section>
 
-  			<section>
-  				<h2>Мы в соцсетях</h2>
-  				<ul>
-  					<li><a href="#">Твиттер</a></li>
-  					<li><a href="#">Фейсбук</a></li>
-  					<li><a href="#">Ютуб</a></li>
-  				</ul>
-  			</section>
+<section>
+  <h2>Заинтересовались?</h2>
+  <p>Укажите предполагаемые даты поездки,<br>
+   и мы покажем вам лучшие предложения гостиниц в седоне  </p>
+  <button>Поиск гостиницы в Седоне</button>
+  <!-- Здесь будет форма поиска гостиниц -->
+  <img src="https://via.placeholder.com/1199x594" alt="Карта поиска гостиниц">
+</section>
+</main>
 
-  			<section>
-  				<p>Разработано <a href="https://htmlacademy.ru">HTML Academy</a></p>
-  			</section>
-  		</footer>
+<footer>
+ <section>
+  <h2>Контакты</h2>
+  <p>#visitSEDONA<br>
+    Тел: <a href="tel:+74956660166">+7 (495) 666-01-66</a>
+  </p>
```

Comment:
Тег параграфа создает перенос строки, потому тут можно обойтись без `<br>`
```suggestion
  <p>#visitSEDONA</p>
    Тел: <a href="tel:+74956660166">+7 (495) 666-01-66</a>
```